lib/ui/map/map_view.dart

```diff
@@ -249,6 +228,9 @@ class MapViewState extends State<MapView> {
   }
 
   Future<LatLng> _getUserPosition() async {
+    if (!(await _locationPermissionGranted())) {
```

Comment:
Only disadvantage of this implementation is that the permission request dialog won't automatically show again on the map page if the user skipped the intro and thus did not previously give location access.

Reply:
Ok, then I'll rework this to ask again for the permission. I was just trying to get rid of all the exceptions :D